lib/pages/authentication/forms/MiscForm.dart

```diff
@@ -61,7 +63,7 @@ class MiscFormState extends State<MiscForm> {
               BlankPadding(),
               getSemester(),
               BlankPadding(),
-              getGrades(),
+              currentSemester != "1" ?  getGrades() : BlankPadding(),
```

Comment:
If the semester is the 1st it throws some errors because the validation expects a pdf